src/query/validation.c

```diff
@@ -401,18 +402,34 @@ void collect_equalities_from_filters(Node *node, List **subjects, List **targets
   FAILWITH("Only equalities between generalization expressions and constants are allowed as pre-anonymization filters.");
 }
 
-static void verify_where(Query *query)
+static Var *get_bucket_expression_column_ref(Node *node)
```

Comment:
The arg tells nothing about the input. I would prefer a description to give some info what this func does.